src/gpl/test/gpl_aux.py

```diff
@@ -0,0 +1,195 @@
+import utl
+
+# Definitions
+DEFAULT_TARGET_DENSITY = 0.7
+
+
+# Besides design, there are no positional args here. General strategy is that 
+# when an arg is None, we just skip setting it, otherwise we will set the
+# parameter after a quick type check.
+def global_placement(design, *,
```

Comment:
Most test cases pass 1-4 flags.  Couldn't we just have them called in-situ rather than using this big wrapper?

Reply:
We can certainly do that. One of the reasons that I ported the global_place command because I thought that the defaults set there were not done in the C++ code (well that and the error checking). I'll take a look.

Reply:
Sharing the error checking makes sense though again that should move to c++.

Reply:
I did look at it more. It appears that there are a handful of defaults that are only set in the tcl (and by extension the gpl_aux.py code). So if we got rid of gpl_aux we would need on the order of 8-10 lines of Python in each of the tests. Moreover, it would be inconvenient to update ~20 tests with new defaults. So I say for now, stay with the gpl_aux wrapper.